Resources/views/sckenhancedselection_content_field.html.twig

```diff
@@ -1,7 +1,16 @@
-{% block ngclasslist_field %}
+{% block sckenhancedselection_field %}
+{% set available_options = fieldSettings.options %}
 {% spaceless %}
-    {% for identifier in field.value.identifiers %}
-        {{ identifier }}{% if not loop.last %}, {% endif %}
-    {% endfor %}
+    {% if available_options|length > 0 %}
```

Comment:
Hello @emodric 

In our initial testing we did not find this check as required. Today we realize this was because of our undocumented template code calling the field rendering.

```twig
{% if not ez_is_field_empty(content, "selection") %}
      {{ez_render_field(content, "selection")}}
{% else %}
       Empty selection
{% endif %}
```

See with the above code if field.value.identifiers is empty we would get the output 'Empty selection' instead as the conditional block (above) would never actually run the 'ez_render_field'.

We are modifying our calling template to remove the use of the conditional block (above) and directly just calling 'ez_render_field' during our testing and refactoring to address the issues you point our here.

Cheers,
Brookins Consulting